supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -268,3 +268,23 @@ For example, you might combine existing product name attributes to create compou
 :name-runtime-vertx: Eclipse Vert.x
 :name-spring-reactive: {name-runtime-spring-boot} with {name-runtime-vertx} reactive components
 ----
+
+[[statements-about-the-future]]
+== Statements about the future
+
+=== Making statements about future releases or plans
```

Comment:
Since there's only one section, I don't think you need both of these headings. I'd probably remove this one (or incorporate the wording into the above heading, "Statements about future releases or plans" or something like that)

Reply:
@bergerhoffer Good point. That makes sense. I'll do a quick fix on it right now.

Reply:
@bergerhoffer Just pushed the changes out. PTAL.